R/cube-residuals.R

```diff
@@ -57,7 +57,7 @@ setMethod('rstandard', 'CrunchCube', function (model) {
     if (any(startsWith(getDimTypes(model), "mr_"))) {
         return(standardizedMRResiduals(model))
     } else {
-        return(chisq.test(as.array(model))$stdres)
+        return(chisq.test(as.array(noTransforms(model)))$stdres)
```

Comment:
This nested function call is getting a bit gnarly, it might be worth breaking it up into an intermediary object.